examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -57,18 +67,25 @@ export class TaskList extends DataObject implements ITaskList {
      */
     private readonly tasks = new Map<string, Task>();
 
-    public readonly addTask = (id: string, name: string, priority: number) => {
+    public readonly addTask = (id: string, name: string, priority: number): void => {
         if (this.tasks.get(id) !== undefined) {
             throw new Error("Task already exists");
         }
         const nameString = SharedString.create(this.runtime);
         nameString.insertText(0, name);
-        const priorityCell: SharedCell<number> = SharedCell.create(this.runtime);
+        const priorityCell = SharedCell.create(this.runtime) as SharedCell<number>;
         priorityCell.set(priority);
+
         // To add a task, we update the root SharedDirectory.  This way the change is propagated to all collaborators
         // and persisted.  In turn, this will trigger the "valueChanged" event and handleTaskAdded which will update
         // the this.tasks collection.
-        this.root.set(id, { id, name: nameString.handle, priority: priorityCell.handle });
+        const data: PersistedTask = {
```

Comment:
how come we are adding this in here?

Reply:
oh nvm, I see, it's something we are passing in to the function after, but are declaringbeforehand.

Reply:
Mostly just to ensure we're typing things as expected, since the SharedDirectory takes any value. Adds confidence that we can cast to the right type when reading the data back out.